en/SetupView/Joystick.md

```diff
@@ -50,6 +50,25 @@ The expo slider allows you to make the sticks less sensitive in the center, allo
 
 The higher the Expo value, the flatter the curve is at the center, and steeper it is at the edges.
 
+## Advanced Settings
+
+![Joystick setup - Expo Curve](../../assets/setup/joystick_advanced.jpg)
+
+The advanced settings are usually not recommended for everyday users, as they can cause unpredicted results if changed wrongly. The following settings are available:
+
+- **Joystick Mode**: Changes what the joystick actually controls, and the mavlink messages sent to the vehicle.
+    - **Normal**: User controls as if using a regular RC radio, mavlink [MANUAL_CONTROL](https://mavlink.io/en/messages/common.html#MANUAL_CONTROL) messages are used.
+    - **Attitude**: User controls the vehicle attitude, mavlink [SET_ATTITUDE_TARGET](https://mavlink.io/en/messages/common.html#SET_ATTITUDE_TARGET) messages are used.
+    - **Position**: User controls the vehicle position, mavlink [SET_POSITION_TARGET_LOCAL_NED](https://mavlink.io/en/messages/common.html#SET_POSITION_TARGET_LOCAL_NED) messages with bitmask for **position** only are used.
+    - **Force**: User controls the forces applied to the vehicle, mavlink [SET_POSITION_TARGET_LOCAL_NED](https://mavlink.io/en/messages/common.html#SET_POSITION_TARGET_LOCAL_NED) messages with bitmask for **force** only are used.
+    - **Velocity**: User controls the forces applied to the vehicle, mavlink [SET_POSITION_TARGET_LOCAL_NED](https://mavlink.io/en/messages/common.html#SET_POSITION_TARGET_LOCAL_NED) messages with bitmask for **velocity** only are used.
+
+- **Message Frequency**: This handles how often the joystick messages are sent. The default is 25 Hz, automatically lowered to 5Hz when there are no changes.
+
+- **Enable Circle Correction**: Circle correction is used because joysticks usually don't have as much stick travel as RC radios, these can describe a square, while joysticks usually describe a circle. This options inscribes a square into the joystick movement area. This way it is possible to reach all four corners, but this costs some resolution as the effective stick travel is reduced.
```

Comment:
Disabled: The joystick position is sent to the vehicle the way that it is read from the joystick device, unchanged. On some joysticks, the (roll, pitch) values are confined to the space of a circle inscribed inside of a square. In this figure, point B would command full pitch forward and full roll right, but the joystick is not able to reach point B because the retainer is circular.  This means that you will not be able to achieve full roll and pitch deflection simultaneously.
![](https://ds055uzetaobb.cloudfront.net/image_optimizer/04275a281b8176289a10fd007945dc59204bfeaa.jpg)
Enabled: The joystick values are adjusted in software to ensure full range of commands. The usable area of travel and resolution is decreased, however, because the area highlighted grey in the figure is no longer used.
![](https://ds055uzetaobb.cloudfront.net/image_optimizer/6ffee5b5f582f78d782b61cbdeca791657458164.jpg)

Reply:
make a separate commit. maybe hamish likes less detail